.github/workflows/pr-labeler.yaml

```diff
@@ -25,7 +25,7 @@ run-name: >-
   Label pull request ${{github.event.pull_request.number}} by ${{github.actor}}
 
 on:
-  pull_request:
+  pull_request_target:
```

Comment:
can you leave a comment about why _target and why generally, others should use `pull_request`.

Similarly leave a comment on the permissions that are write with a description of why, this link https://docs.github.com/en/rest/authentication/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-pull-requests and point out that most other users shouldn't need this.

I just wanna make sure this pattern doesn't leak out and people over grant access

Reply:
Good idea! Will do.

Reply:
And thanks for the comment.